owned-manifests/machinehealthcheck.crd.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: machinehealthchecks.healthchecking.openshift.io
+spec:
+  group: healthchecking.openshift.io
+  names:
+    kind: MachineHealthCheck
+    listKind: MachineHealthCheckList
+    plural: machinehealthchecks
+    singular: machinehealthcheck
+  scope: Namespaced
+  version: v1alpha1
```

Comment:
Should not we at least validate the metadata? See https://github.com/openshift/machine-api-operator/blob/master/owned-manifests/machine.crd.yaml#L16-L24